### ISSUE TITLE
Save screen dpi

### ISSUE THREAD

**Issue by alimd** [LeaVerou/dpi#5]
@LeaVerou Nice Project.

What do you think about this feature ? 
Log/Save clients screen resolution and dpi

It can be a great list of devices screen dpi and resolution in future.

**Comment by hugodias** [comment]
+1 to this.

**Comment by LeaVerou** [comment]
Ah, good idea!
Might be a bit tricky to implement w/out server-side code though (it’s hosted on Github Pages).

**Comment by alimd** [comment]
@LeaVerou yes, i think you must have api on another server
for example simple script in nodejs in you server.

**Comment by alimd** [comment]
Can we use github api to store in a gist ?! its not slow in transaction or ...

**Comment by LeaVerou** [comment]
Yes, that’s how Dabblet works too. But it would need a good bit of work.

**Comment by alimd** [comment]
any plan for migrate to another server or close this issue?
